README.md

```diff
@@ -110,11 +110,7 @@ data Teletype m a where
   ReadTTY  :: Teletype m String
   WriteTTY :: String -> Teletype m ()
 
-readTTY :: Eff Teletype m => m String
-readTTY = send ReadTTY
-
-writeTTY :: Eff Teletype m => String -> m ()
-writeTTY str = send (WriteTTY str)
+makeEff ''Teletype
```

Comment:
Please revert the changes to the README: `TemplateHaskell` is not part of the required language extensions as listed in the README, and I want these code examples to work without enabling any additional extensions. I do not want to list `TemplateHaskell` as a required extension.